### PR TITLE
Fix the func to check the ip6tables rules

### PR DESCRIPTION
**Pull request by yalzhang**
iface_network.py: in the former version, it will check the ipv4 rules
for 2 times. Edit the function to check ipv4 and ipv6 each and update
some rules.

Signed-off-by: yalzhang <yalzhang@redhat.com>

### CODE REVIEW

**Comment by yalzhang** [comment]
The related cases are virtual_network.iface_network.net_forward.. and run successfully locally.

$ run iface_network
JOB ID     : e69aaab5b51fd996985b21a8025ea45c0b90f7ce
JOB LOG    : /root/avocado/job-results/job-2019-03-05T01.24-e69aaab/job.log
 (01/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.bridge_test: PASS (50.03 s)
 (02/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.dnsmasq_test.net_dns_forward: PASS (52.36 s)
 (03/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.dnsmasq_test.net_domain: PASS (52.15 s)
 (04/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.dnsmasq_test.net_host_ip: FAIL: Can't resolve name foo.example.com on guest (57.79 s)
 (05/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.dnsmasq_test.net_dhcp_range: PASS (176.22 s)
 (06/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.dnsmasq_test.net_dns_host: PASS (58.00 s)
 (07/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.qos_test.qos_option_bandwidth.positive_test: PASS (55.57 s)
 (08/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.qos_test.qos_option_bandwidth.negtive_start_test: PASS (10.82 s)
 (09/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.qos_test.qos_option_bandwidth.negtive_remove_test: PASS (56.38 s)
 (10/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.net_forward.net_isolated: PASS (77.05 s)
 (11/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.net_forward.net_nat: PASS (78.16 s)
 (12/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.net_forward.net_route: PASS (77.49 s)
 (13/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.net_forward.net_open.default: PASS (67.96 s)
 (14/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.net_forward.net_open.without_bridge: PASS (68.45 s)
 (15/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.net_forward.net_open.without_mac: PASS (68.42 s)
 (16/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.net_forward.net_open.negative_without_ip: PASS (8.90 s)
 (17/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.net_forward.net_open.negative_with_dev: PASS (8.98 s)
 (18/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.net_forward.route_test: PASS (69.49 s)
 (19/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.net_pxe: CANCEL: Please provide initrd/vmlinuz URL (7.49 s)
 (20/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.net_guest: PASS (228.07 s)
 (21/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.net_macvtap.error_test.start_error: PASS (9.66 s)
 (22/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.net_macvtap.error_test.define_error: PASS (8.94 s)
 (23/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.net_macvtap.net_bridge.default_test: PASS (64.18 s)
 (24/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.net_macvtap.net_bridge.multi_guests: PASS (132.51 s)
 (25/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.net_macvtap.net_passthrough: PASS (63.15 s)
 (26/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.net_macvtap.net_private: |^PASS (63.12 s)
 (27/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.net_portgroup.default_portgroup: PASS (65.65 s)
 (28/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.net_portgroup.specific_portgroup: PASS (69.34 s)
 (29/29) type_specific.io-github-autotest-libvirt.virtual_network.iface_network.net_portgroup.overwriting_bandwidth: PASS (69.33 s)
RESULTS    : PASS 27 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 1
JOB TIME   : 1883.21 s

File: libvirt/tests/src/virtual_network/iface_network.py [hunk 1]
```diff
@@ -398,6 +401,13 @@ def check_ipt_rules(check_ipv4=True, check_ipv6=False):
                         test.fail("Can't find iptable rule:\n%s" % ipt)
             return ipv4_rules
         if check_ipv6:
+            ipt_rules = (
+                "INPUT -i %s -p udp -m udp --dport 53 -j ACCEPT" % br_name,
+                "INPUT -i %s -p tcp -m tcp --dport 53 -j ACCEPT" % br_name,
+                "INPUT -i %s -p udp -m udp --dport 547 -j ACCEPT" % br_name,
+                "FORWARD -i {0} -o {0} -j ACCEPT".format(br_name),
+                "FORWARD -o %s -j REJECT --reject-with icmp" % br_name,
+                "FORWARD -i %s -j REJECT --reject-with icmp" % br_name)
```
Comment (chloerh) [inline libvirt/tests/src/virtual_network/iface_network.py]: Duplicated lines, should do something about it

Comment (yalzhang) [reply libvirt/tests/src/virtual_network/iface_network.py]: I have updated it, please help to review, Thank you!

**Comment by chloerh** [comment]
LGTM

**Pull request merged**